ruby_challenges.rb

```diff
@@ -0,0 +1,112 @@
+# ASSESSMENT 5: Ruby Coding Practical Questions
+# MINASWAN
+
+## I AM WRITNG PSEUDO CODE ON THIS CHALLENGE BECAUSE I FOUND IT HELPS ME A LITTLE MORE WHEN I DO.##
+
+# --------------------1) Create a method that takes in an array of words and a single letter and returns all the words that contain that particular letter.
+
+beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+
+letter_o = 'o'
+# Expected output: ['coffee', 'soda water']
+letter_t = 't'
+# Expected output: ['tea', 'water', 'soda water']
+
+def beverages(array, letter)
+  array.select do |value|
+ value.include?(letter)
+  end
+end 
+p beverages beverages_array, letter_o
+p beverages beverages_array, letter_t
+
+# PSEUDO CODE
+# Create a method that takes in an array of words and a single letter and returns all the words that contain that particular letter
+# I will use .select to iterate through my array and find the words containing 'o' and 't' it works like a the .filter hof
+# I will use the |value| extenstion of .select that will call on every value of the array to look for my filter requests as it loops through the array
+# After we filter and go through each value, we ask that the values included in our search search for a letter of iether o or t
+#  by definetley including the search with the include? we are ensuring we do not get a return on anything other than what we ask for which in this case is 'o' or 't'
+# 
+
+# -------------------2) Create a method that takes in an array of numbers and returns the sum of the numbers. Use the test variables provided.
+
+nums_array1 = [42, 7, 27]
+# # # Expected output: 76
+
+nums_array2 = [25, 17, 47, 11]
+# # Expected output: 100
+
+def sum(array)
+  array.sum
+
+end
+
+p sum nums_array1
+p sum nums_array2
```

Comment:
I would recommend naming your method something that is not a ruby built-in method.  In a larger application, that may cause conflicts.